### ISSUE TITLE
5.0.59 not installing on kodi

### ISSUE THREAD

**Issue by MARKJ34** [Tempest0580/tempest#47]
hi
i have a dual boot core elec kodi 18.5 / Android kodi 18.6 unit. on both installs Tempest 5.0.59 will not install it says all dependencies fail to install, Even tho I have auto update switched off the new version tries to auto install and when it doesn't it removes the older version under core elec, but not under Android for some reason. 
Anyway, neither install.....
any ideas? regards Mark

**Comment by Tempest0580** [comment]
Have you tried to restart kodi? Or the device? Sometimes that works.

**Comment by MARKJ34** [comment]
yes many times, on both builds. fails to install every time.  can you re test your install on your dev box?

**Comment by Tempest0580** [comment]
I'll look into it.

**Comment by anthob1** [comment]
Same issue here. Occurs with attempted auto installs of Tempest, TempTV, and Arconai on android Kodi 18.6, just as described by Markj34 above.

**Comment by Tempest0580** [comment]
Try installing the dependencies separately. 
When you try to install tempest it will pop up and show all the dependencies that will be installed.  Go over and try to install them from their.

**Comment by MARKJ34** [comment]
hi ok but can you plz list here the full list of those dependencies that need to be installed. and do you do that before you try to update tempest? can i ask does it install ok for you on your dev box?

**Comment by anthob1** [comment]
Tried to install Js2py 0.0.0, tzlocal 1.5.1, pytz 2014.2 separately from TempTV repo 0.6.6. All failed. All other dependencies are either listed as 'installed' or 'optional.'

**Comment by Tempest0580** [comment]
Sorry been busy, Working on a few things plus back to work full time.
I have not had any issues installing Tempest even on a fresh install But im still looking into it.
Thanks

**Comment by anthob1** [comment]
I think I might have stumbled onto a solution. I had to install the dependencies in sequence. I believe the order was pytz, then tzlocal, then Js2py. Check the dependencies 'within' the dependencies. After that, I was able to update the addons.

QUESTION: I noticed that there is a newer version  of tzlocal in the kodi repo. Should I install that or just stick with version 1.5.2 from the Tempest repo?

Side note: Glad to hear that Tempest0580 is working full time again. Thanks for your projects.
